web_accounts_list.json

```diff
@@ -339,6 +339,15 @@
          "category" : "shopping",
          "valid" : true
       },
+      {
+         "name" : "Ko-Fi",
+         "check_uri" : "https://ko-fi.com/{account}",
+         "account_existence_code" : "200",

```

Comment:
Thanks for the submission. For this entry to be included in the project, I need you to find the `account_existence_string` (something unique to the page where an account is) and the `account_missing_string` (something specific in the response from the server when an invalid account was requested).

Reply:
There you go. I left them out initially simply because non-existent profiles always result in 302. I added in the existing and missing strings. Thanks!